recipe/meta.yaml

```diff
@@ -29,7 +29,10 @@ outputs:
         - jupyterlab_server
```

Comment:
Shouldn't those requirements be identical to https://github.com/mamba-org/gator/blob/master/setup.py#L75 ?

Reply:
Thank you for quickly looking at this. Yes that is the plan. In practice, it does not change anything as those libraries are dependencies of already set requirements (except for requests that need to be dropped). This is more to remove the warning from the bot (and because it is totally right that explicitly used libraries should appear in the requirements).

Reply:
PR is opened: https://github.com/mamba-org/gator/pull/124

Reply:
Nice... Still wondering why `pip check` is not complaining in that case.

Reply:
From my understanding pip checks that the requirements are fulfilled. In this case they are. The wrongly added library `requests` is simply ignored and the 3 libraries that I'm expliciting with this PR got installed as they are dependencies of `notebook`.

This PR will clean the dependency analysis done by the autotick bot (see for example https://github.com/conda-forge/jupyter_conda-feedstock/pull/20).